platform/mellanox/mlnx-platform-api/sonic_platform/fan.py

```diff
@@ -22,16 +22,23 @@
 
 FAN_PATH = "/var/run/hw-management/thermal/"
 LED_PATH = "/var/run/hw-management/led/"
+# fan_dir only exist SPC2 switches
+FAN_DIR = "/var/run/hw-management/system/fan_dir"
+
+skus_with_fan_dir = {'ACS-MSN3700', 'ACS-MSN3700C', 'ACS-MSN3800'}
```

Comment:
you should ask for a way to get the capability from the driver. if it supports it or not.
you should not have hard coded values for specific SKUs.
next it will be Spectrum-3 and it will be supported.
Soon we need to support 'B' SKUs and this means you will need to extend this list as well.

if this exists in other places we need to change the logic there as well

Reply:
Currently, fan direction isn't supported on spc1. So I'm going to use chip type to test.